internal/lang/funcs/filesystem_test.go

```diff
@@ -217,6 +217,63 @@ func TestTemplateFile(t *testing.T) {
 	}
 }
 
+func TestTemplateString(t *testing.T) {
+	tests := []struct {
+		Data  cty.Value
+		Vars  cty.Value
+		Want  cty.Value
+		Err   string
+	}{
```

Comment:
It would be nice if the tests each had a name describing what it's testing.

Reply:
commit [4fb1950](https://github.com/opentofu/opentofu/pull/1223/commits/4fb1950bb377f66c61af3af40afb85f8d11d343a) resolves this